Tests/Functions/Public/Show-Karma.Tests.ps1

```diff
@@ -217,5 +217,28 @@ Describe 'Show-Karma' {
                 }
             }
         }
+
+        Context 'With -Meditate Switch' {
+
+            Context 'PSKoans directory exists' {
+                BeforeAll {
+                    Mock Get-PSKoanLocation { "$HOME/PSKoans" }
+                }
+
+                It 'should find the PSKoan directory' {
+                    Test-Path -Path (Get-PSKoanLocation) | Should -Be $true
```

Comment:
Ah, I see where the misunderstanding is. Np, np. 🙂 

OK, so what you need to do is mock `Test-Path` , `Invoke-Item`, and `Get-Command` above, and then test the `Show-Karma -Meditate` switch by calling that command directly. I think one of the existing tests already has a section for the `-Meditate` switch, which you can literally just move down to this section under the `directory exists` context. Then all you need to do is copy that logic and adjust it for the case where the directory doesn't exist. You'd need to mock `Update-PSKoan` for that case as well, since all we need to verify is that that command gets called.

After all, the purpose of the test is to show that `Show-Karma` is behaving correctly, which we can't be sure of until we actually call the command. 🙂 

If any of that isn't clear just let me know, I hope I'm not leaving you to fumble about in the dark too much! 😊